h2o-algos/src/main/java/hex/generic/GenericModelOutput.java

```diff
@@ -17,6 +19,7 @@
     final ModelCategory _modelCategory;
     final int _nfeatures;
     TwoDimTable _variable_importances;
+    TwoDimTable _model_parameters;
```

Comment:
Having a universal structure is fine to me, but let's discuss about the reasoning. Having one `GenericModelParameters` per algorithm introduces a lot of new objects, but the clients should be able to work with such parameters object without any adjustments. Have you tried that ? Downside of that is a lot of new objects introduced (which I did with Metrics, which are slightly more complex objects and it might change in future).

This obviously leads to params being deserialized as list instead of a dict in Python. Which changes the experience for our users. Not sure about R.